Mathlib/Tactic/ToAdditive/Frontend.lean

```diff
@@ -1287,217 +1299,6 @@ partial def addToAdditiveAttr (src : Name) (cfg : Config) (kind := AttributeKind
 
 end
 
-/--
```

Comment:
I see you deleted a number of comments, like this one. Is all the information still documented somewhere?

So long as we're not losing documentation (or if what you do delete you think is not relevant anymore), then the PR looks good to me.

bors d+